app/gateway/changelog.md

```diff
@@ -9,6 +9,66 @@ Changelog for supported Kong Gateway versions.
 
 For product versions that have reached the end of sunset support, see the [changelog archives](https://legacy-gateway--kongdocs.netlify.app/enterprise/changelog/).
 
+
+## 3.6.1.2
+**Release Date** 04/05/2024
+
+### Features
+
+#### Plugins
+
+* [**OAS Validation**](/hub/kong-inc/oas-validation/) (`oas-validation`)
+  * Added the new field `api_spec_encoded` to indicate whether the `api_spec` is URI-encoded.
+
+### Fixes
+
+#### Clustering
+
+* Modified the current AWS Vault backend to support `CredentialProviderChain` so that users can choose not 
```

Comment:
```suggestion
* Adjusted clustering compatible check related to AWS Secrets Manager.
```

@lena-larionova there is a changelog entry update: https://github.com/Kong/kong-ee/compare/0781fb36fdf5f6872494640bb3e0b277a86d8627..6e4f3e7231e78440d97cf0e425f501742463be0d